src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead tr');
+const tbody = table.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const number = string => string.replace(/[$,]/g, '');
```

Comment:
Consider renaming the 'number' function to a more descriptive name, e.g. 'formatSalary'.